specification/cognitiveservices/data-plane/Face/stable/v1.0/examples/CreateNewFaceList.json

```diff
@@ -6,7 +6,8 @@
     "faceListId": "sample_face_list",
     "body": {
       "name": "sample_list",
-      "userData": "User-provided data attached to the face list"
+      "userData": "User-provided data attached to the face list",
+      "RecognitionModel": "Recognition model used for the face list"
```

Comment:
I think the validation will fail for this. We should use `recognition_v02` here, same for others.

Reply:
Done.